### PR TITLE
[W11-3] TravelBuddy

### PR DESCRIPTION
**Pull request by chung-ming**

### CODE REVIEW

**Comment by kylase-learning** [comment]
## Feedback (Week 7; v1.1)
 
### Github Project Page
- [x] Update Project description
- [x] Update Link to Project Page
 
### README
- [x] UI Mockup updated in project website
- [ ] Make the project website look like a product home page (aka remove the top nav bar)
- [x] AB4 acknowledgement
- [x] Travis badge updated to own fork's
- [x] Coveralls badge updated to own fork's
 
### [Project Page](https://cs2103-ay1819s2-w11-4.github.io/main/)
- [x] Update to remove AB4 reference
- [x] Travis badge updated to own fork's
- [x] Coveralls badge updated to own fork's

### About Us Page
- [x] Updated with photos
- [x] Roles and components assigned

UG/DG feedback will be provided to you in the documents.

Generally, I can see that all of you do not have huge issue with the workflow. Keep up the good work! 👍

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -2,60 +2,64 @@
...
+            + PREFIX_DESCRIPTION
+            + "The National University of Singapore is an autonomous research university in Singapore. "
+            + PREFIX_ADDRESS + "21 Lower Kent Ridge Road, Singapore 119077 "
+            + PREFIX_TAG + "school "
+            + PREFIX_TAG + "studiesHere";
 
-    public static final String MESSAGE_SUCCESS = "New person added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book";
+    public static final String MESSAGE_SUCCESS = "New place added: %1$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This place already exists in the address book";
```
Comment (kylase-learning) [inline src/main/java/seedu/address/logic/commands/AddCommand.java]: The constant name should be `MESSAGE_DUPLICATE_PLACE`? 

Let's try to fix all of these 😄

File: src/main/java/seedu/address/logic/commands/DeleteCommand.java [hunk 2]
```diff
@@ -9,21 +9,21 @@
...
-
+    public static final String COMMAND_ALIAS = "d";
     public static final String MESSAGE_USAGE = COMMAND_WORD
-            + ": Deletes the person identified by the index number used in the displayed person list.\n"
+            + ": Deletes the place identified by the index number used in the displayed place list.\n"
             + "Parameters: INDEX (must be a positive integer)\n"
             + "Example: " + COMMAND_WORD + " 1";
 
-    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Person: %1$s";
+    public static final String MESSAGE_DELETE_PERSON_SUCCESS = "Deleted Place: %1$s";
```
Comment (kylase-learning) [inline src/main/java/seedu/address/logic/commands/DeleteCommand.java]: Should be `MESSAGE_DELETE_PLACE_SUCCESS`?

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 3]
```diff
@@ -20,88 +21,93 @@
...
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " 1 "
-            + PREFIX_PHONE + "91234567 "
-            + PREFIX_EMAIL + "johndoe@example.com";
+            + PREFIX_COUNTRY_CODE + "SGP "
+            + PREFIX_RATING + "4 "
+            + PREFIX_DESCRIPTION + "No description";
 
-    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Person: %1$s";
+    public static final String MESSAGE_EDIT_PERSON_SUCCESS = "Edited Place: %1$s";
```
Comment (kylase-learning) [inline src/main/java/seedu/address/logic/commands/EditCommand.java]: Should be `MESSAGE_EDIT_PLACE_SUCCESS`?

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 1 above)
Comment (chung-ming) [reply src/main/java/seedu/address/logic/commands/AddCommand.java]: Okay

**Pull request closed without merging**